lib/generator.js

```diff
@@ -38,7 +38,6 @@ var ProjectType = {
   wap: "WAP",
   website: "WEBSITE",
   node: "NODE",
-  ruby: "RUBY",
```

Comment:
I just ran a `build`, as you can see in `generator._js` there was no such change